values.schema.json

```diff
@@ -0,0 +1,479 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "https://coder.com/coderhelm.schema.json",
+    "title": "Coder Enterprise Helm Files",
+    "description": "TODO",
+    "type": "object",
+    "additionalProperties": false,
+    "$comment": "Set all the required fields",
+    "required": [],
+    "properties": {
+        "storageClassName": {
+            "type": "string",
+            "$comment": "TODO: Is this always a string? Is null an ok field? Can it be omitted?",
+            "default": ""
+        },
+        "serviceType": {
+            "type": "string",
+            "default": "ClusterIP",
+            "$comment": "TODO: Check the enum list is complete",
+            "enum": [
+                "ClusterIP",
+                "NodePort",
+                "LoadBalancer",
+                "ExternalName"
```

Comment:
ExternalName isn't supported by our chart because you need to set extra fields for it.